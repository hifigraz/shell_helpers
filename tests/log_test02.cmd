logDebug "ASDF"
setLogLevel ${LVL_DEBUG}
logDebug "ASDF"
logTrace "ASDF"
setLogLevel ${LVL_TRACE}
logTrace "ASDF"
setLogLevel ${LVL_INFO}
logTrace "ASDF"
logDebug "ASDF"
logTrace "ASDF"
logDebug "ASDF"
logCritical "ASDF"
logWarning "ASDF"

