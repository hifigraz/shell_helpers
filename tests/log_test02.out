00000000:000000 DEBUG    ASDF
00000000:000000 TRACE    ASDF
00000000:000000 CRITICAL ASDF
00000000:000000 WARNING  ASDF
